src/utils/_variables.scss

```diff
@@ -0,0 +1,16 @@
+$day-size: 100px;
+$day-count: 31;
+$border-size : $day-size / 100;
+$gap-size: $day-size / 100;
+$mockup-padding: $day-size / 10;
+$calendar-height: 100vh;
```

Comment:
currently I don't see the usage so I have no idea of limitations of this height
That's why I would prefer not to use variable here at all

Reply:
Good advice ! Agree wtih your approach . 
Formulas was add  to automate all was it possible for experemental purpose . Formulas confuse and  reduce redability of code and harden understanding of limitations in that case.

Reply:
I am not about a formula but about `100vh` in a variable

Reply:
regarding $calendar-height: 100vh - I was excited by possbilities of sass and tried to implemet as many variables, extends , mixins as possible . For sure - that has to be a direct setting of the field.